src/pages/login/login.js

```diff
@@ -0,0 +1,93 @@
+import { Link, useNavigate } from "react-router-dom";
+
+import axios from "axios";
+import { useState } from "react";
+import "./login.css";
+import { useAuth } from "../../contexts/auth-context";
+const Login = () => {
+  const navigate = useNavigate();
+  const { setAuthState } = useAuth();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const callUserLogin = async (emailId, passw) => {
+    try {
+      const { data } = await axios.post("/api/auth/login", {
+        email: emailId,
+        password: passw,
+      });
+      localStorage.setItem("token", data.encodedToken);
+      localStorage.setItem("userName", data.foundUser.firstName);
+      console.log(localStorage.token, localStorage.userName);
+      setAuthState({
+        isAuth: localStorage.token ? true : false,
+        userInfo: localStorage.userName ? localStorage.userName : null,
+      });
+      navigate("/");
+    } catch (error) {
+      console.log(error);
```

Comment:
Instead of console.log can use throw(error)

Reply:
Yes okk thanks